app/views/images/new.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= form_for Image.new, html: {multipart: true} do |i| %>
```

Comment:
Ah, this is why you didn't specify `@image` earlier. Pass data to your views through your controllers